<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GUIClass</class>
 <widget class="QMainWindow" name="GUIClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GUI</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="0" column="0">
     <widget class="QGroupBox" name="ParametersGroupBox">
      <property name="title">
       <string>Parameters</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="3" column="1">
        <widget class="QLineEdit" name="MarginalAdhesionLineEdit"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="CellSizeLabel">
         <property name="text">
          <string>Uniformity of Cell Size</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="CellSizeLineEdit"/>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="ClumpThicknessLabel">
         <property name="text">
          <string>Clump Thickness</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="MarginalAdhesionLabel">
         <property name="text">
          <string>Marginal Adhesion</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="CellShapeLabel">
         <property name="text">
          <string>Uniformity of Cell Shape</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="CellShapeLineEdit"/>
       </item>
       <item row="6" column="1">
        <widget class="QLineEdit" name="BlandChromatinLineEdit"/>
       </item>
       <item row="8" column="0">
        <widget class="QLabel" name="MitosesLabel">
         <property name="text">
          <string>Mitoses</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QLineEdit" name="BareNucleiLineEdit"/>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="NormalNucleoliLabel">
         <property name="text">
          <string>Normal Nucleoli</string>
         </property>
        </widget>
       </item>
       <item row="8" column="1">
        <widget class="QLineEdit" name="MitosesLineEdit"/>
       </item>
       <item row="4" column="1">
        <widget class="QLineEdit" name="EpithelialCellSizeLineEdit"/>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="BlandChromatinLabel">
         <property name="text">
          <string>Bland Chromatin</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="ClumpThicknessLineEdit"/>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="BareNucleiLabel">
         <property name="text">
          <string>Bare Nuclei</string>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QLineEdit" name="NormalNucleoliLineEdit"/>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="EpithelialCellSizeLabel">
         <property name="text">
          <string>Single Epithelial Cell Size</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QFrame" name="PushButtonsFrame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QPushButton" name="RunProcessPushButton">
         <property name="text">
          <string>Run Process</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QFrame" name="LogsFrame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="LogsLabel">
         <property name="font">
          <font>
           <pointsize>20</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Logs</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="ClearLogsPushButton">
         <property name="text">
          <string>Clear Logs</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QTextEdit" name="LogsTextEdit">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>18</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>ClumpThicknessLineEdit</tabstop>
  <tabstop>CellSizeLineEdit</tabstop>
  <tabstop>CellShapeLineEdit</tabstop>
  <tabstop>MarginalAdhesionLineEdit</tabstop>
  <tabstop>EpithelialCellSizeLineEdit</tabstop>
  <tabstop>BareNucleiLineEdit</tabstop>
  <tabstop>BlandChromatinLineEdit</tabstop>
  <tabstop>NormalNucleoliLineEdit</tabstop>
  <tabstop>MitosesLineEdit</tabstop>
  <tabstop>RunProcessPushButton</tabstop>
  <tabstop>LogsTextEdit</tabstop>
  <tabstop>ClearLogsPushButton</tabstop>
 </tabstops>
 <resources>
  <include location="GUI.qrc"/>
 </resources>
 <connections/>
</ui>
